app/lib/methods/subscriptions/room.js

```diff
@@ -78,8 +78,11 @@ export default class RoomSubscription {
 	};
 
 	handleConnection = () => {
+		const _lastOpen = this.lastOpen;
 		reduxStore.dispatch(clearUserTyping());
 		RocketChat.loadMissedMessages({ rid: this.rid }).catch(e => console.log(e));
```

Comment:
You can wrap this in a `try/catch`, call it with `async/await` instead of `then/catch` and call `read` after it's resolved.